.npmignore

```diff
@@ -0,0 +1,41 @@
+.idea/
```

Comment:
We're sorting .ignore lists alphabetically. This list is sorted by folder/file, then alphabetically.